platform/web-security/src/main/java/org/motechproject/security/passwordreminder/PasswordExpirationChecker.java

```diff
@@ -0,0 +1,56 @@
+package org.motechproject.security.passwordreminder;
+
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceReference;
+import org.osgi.util.tracker.ServiceTracker;
+import org.osgi.util.tracker.ServiceTrackerCustomizer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+/**
+ * Waits for the scheduler module to start and then schedules password expiration checker job.
```

Comment:
we should add some sort of explanation in this class and the following one, about why they are necessary (the scheduler is not a platform bundle, etc.)